src/devices/machine/cat702.h

```diff
@@ -42,4 +46,14 @@ class cat702_device : public device_t
 	devcb_write_line m_dataout_handler;
 };
 
-#endif // MAME_SONY_CAT702_H
+class cat702_piu_device : public cat702_device
+{
+public:
+	cat702_piu_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock)
+		: cat702_device(mconfig, CAT702_PIU, tag, owner, clock) {}
```

Comment:
Please don’t inline this stuff in headers.  It hurts performance downstream, and you don’t want it to be instantiated in every translation unit that constructs the class.